src/docs/markdown/caddyfile/common-uses.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Common Caddyfile Uses
+---
+
+# Common Caddyfile Uses
+
+This page presents some complete, minimal Caddyfile configurations for common use cases, which you might find helpful when building your own configs.
+
+These are not drop-in solutions; you will have to customize your domain name, ports/sockets, directory paths, etc. They are intended to illustrate only the basic requirements for the most common use cases.
```

Comment:
These don't really involve "requirements" IMO, I think this might be better?

```suggestion
These are not drop-in solutions; you will have to customize your domain name, ports/sockets, directory paths, etc. They are intended to illustrate some of the most common configuration patterns.
```